community/radare2.hpkg

```diff
@@ -0,0 +1,105 @@
+(use ../prelude)
+(import ../core)
+(use ./capstone)
+(use ./file)
+(use ./xxhash)
+(use ./zlib)
+(use ./libzip)
+(use ./lz4)
+(use ./openssl)
+(use ./libuv)
+
+# XXX tree-sitter can be used to enable features.
+
+(defsrc radare2-src
+  :file-name
+  "radare2-4.4.0.tar.gz"
+  :url
+  "https://github.com/radareorg/radare2/archive/4.4.0.tar.gz"
+  :hash
+  "sha256:401ab89f39f7e70e03336f6890dc4fdf52dae4f61201e4cd8a1fbe5a4b9eb451")
+
+(defpkg radare2
+  :builder
+  (fn []
+    (os/setenv "PKG_CONFIG_PATH"
+               (string
+                (join-pkg-paths ":" "/lib64/pkgconfig"
+                                [libuv libzip])
+                ":"
+                (join-pkg-paths ":" "/lib/pkgconfig"
+                                [capstone zlib openssl xxhash lz4])))
+
+    (os/setenv "PATH"
+               (join-pkg-paths ":" "/bin"
+                               [core/coreutils
+                                core/findutils
+                                core/gcc
+                                core/grep
+                                core/make
+                                core/patch
+                                core/sed
+                                core/awk
+                                core/tar
+                                core/pkgconf
+                               ]))
+    (core/link-/bin/sh)
+    (unpack-src radare2-src)
+    (os/setenv "CFLAGS"
+               (string *default-cflags*
+                       " "
+                       (string/join (map |(string "-I" ($ :path) "/include")
+                                         [capstone
+                                          file
+                                          zlib
+                                          lz4
+                                          xxhash
+                                          openssl
+                                          libuv
+                                          libzip]) " ")
+                       " "
+                       (sh/$<_ pkg-config --cflags capstone "liblz4" zlib libuv libxxhash openssl)))
+    (os/setenv "LDFLAGS"
+               (string *default-ldflags*
+                       " "
```

Comment:
I noticed the build results used `RPATH` so I inserted a space-padded `-Wl,--enable-new-dtags` around here.

The result was that `RUNPATH` was used in the resulting executables and shared objects.

Reply:
Fixed in 8bf0901